core/App/screens/ProofDetailsHistory.tsx

```diff
@@ -0,0 +1,100 @@
+import type { StackScreenProps } from '@react-navigation/stack'
+
+import { ProofExchangeRecord } from '@aries-framework/core'
+import { useAgent, useConnectionById, useProofById } from '@aries-framework/react-hooks'
+import { StackNavigationProp } from '@react-navigation/stack'
+import React, { useEffect, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+
+import { GroupedSharedProofDataItem } from '../../verifier/types/proof'
+import { markProofAsViewed } from '../../verifier/utils/proof'
+import SharedProofData from '../components/misc/SharedProofData'
+import { useTheme } from '../contexts/theme'
+import { ProofRequestsStackParams, Screens } from '../types/navigators'
+
+type ProofDetailsProps = StackScreenProps<ProofRequestsStackParams, Screens.ProofDetailsHistory>
+
+interface ProofDetailsHistoryProps {
+  record: ProofExchangeRecord
+  navigation: StackNavigationProp<ProofRequestsStackParams, Screens.ProofDetailsHistory>
+}
+
+const connectionLessLabel = 'Anonymous'
+
+const VerifiedProof: React.FC<ProofDetailsHistoryProps> = ({ record, navigation }) => {
+  const { t } = useTranslation()
+  const { ColorPallet } = useTheme()
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+    },
+    content: {
+      flexGrow: 1,
+      marginHorizontal: 30,
+      marginTop: 10,
+    },
+    descriptionContainer: { marginVertical: 20 },
+    descriptionText: {
+      color: ColorPallet.grayscale.white,
+    },
+    label: {
+      fontWeight: 'bold',
+    },
+  })
+
+  const connection = record.connectionId ? useConnectionById(record.connectionId) : undefined
+  const connectionLabel = useMemo(
+    () => (connection ? connection?.alias || connection?.theirLabel : connectionLessLabel),
+    [connection]
+  )
+
+  const [sharedProofDataItems, setSharedProofDataItems] = useState<GroupedSharedProofDataItem[]>([])
+
+  function onSharedProofDataLoad(data: GroupedSharedProofDataItem[]) {
```

Comment:
change to arrow function to match others

Reply:
fixed